wegodang/products/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+
+from products.views import ProductListView
+
+urlpatterns = [
+    path('/list', ProductListView.as_view())
```

Comment:
8000/products 만으로도 충분히 product list를 가르키는 endpoint의 의미를 갖습니다.

```suggestion
    path('', ProductListView.as_view())
```